x-pack/platform/test/functional/apps/aiops/cases/log_pattern_analysis_cases.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../api_integration/services/ml/security_common';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
```

Comment:
As discussed, would it be better to move these tests up a level, for example by creating `x-pack/platform/test/functional/apps/cases_attachments` to distinguish ownership of these files from `/x-pack/platform/test/functional/apps/ml`  and `/x-pack/platform/test/functional/apps/aiops`?